---
title: "Analyze sweet corn RNAseq data"
author: "Seren Villwock"
date: '2022-06-22'
output: html_document
---

### Load packages (make sure you install them first)
```{r Script setup, echo=FALSE, message=FALSE, echo=FALSE}
library(tidyverse)
library(ggplot2)
library(here)
```


### Script setup

Set file locations relative to the project folder using `here`  
```{r set here path, include=FALSE}
here::i_am("analysis/Analyzing_RNAseq.Rmd")
```

Formatting (you can ignore this unless you want to change something about the formatting)
```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=TRUE)
knitr::opts_chunk$set(message=FALSE)
knitr::opts_chunk$set(cache=FALSE)
knitr::opts_chunk$set(tidy=TRUE)
knitr::opts_chunk$set(results="hold")
knitr::opts_chunk$set(collapse=TRUE)
knitr::opts_chunk$set(fig.show="hold")
knitr::opts_chunk$set(tidy.opts = list(wrap=FALSE))
knitr::opts_knit$set(root.dir = here())
```




## Read in the data: (located in the `data/` folder)
* a .txt file with gene counts from RNAseq
* a .csv with carotenoid data 
* a .csv with a sample naming key

```{r read in data}

```

## Inspect the data
You can do this in your console if you want so it doesn't print to the screen or to the knitted html
Try the functions head(), nrow() & ncol(), colnames()


## Match the carotenoid data sample names with the gene expression data sample names
Use the sample naming key to match the `rlog_ids` to the `accession_name_carot`
(hint: try ?left_join()  )
```{r}

```



